atlas_placement_hints/layered_atlas.py

```diff
@@ -213,16 +219,23 @@ def create_layer_meshes(self, layered_volume: NDArray[np.integer]) -> List["trim
 
         return meshes
 
-    def _compute_dists_and_obtuse_angles(self, volume, direction_vectors):
+    # pylint: disable=W0613
```

Comment:
can you use the name: `unused-argument`; I don't know the numbers off the top of my head

also, can it be scoped to only the functions that need it, instead of it being ignored for the rest of the file

Reply:
Ugh I tried `unused-arguments` before, but couldn't get it to work without resorting to the ID number. Turns out I just needed to leave off the `s` 🤦 . Done. Also rescoped it.